src/pages/home/components/LandingImage.js

```diff
@@ -27,7 +27,11 @@ export const LandingImage = () => {
             Gado-Gado Indonesian Student Association <br />
             of UBC
           </h1>
-          <CustomButton className="grid justify-center" />
+          <Button
+            className="grid justify-center"
+            text="Become a member!"
+            landingButton={true}
```

Comment:
Why don't we just directly use the landingButton component?

Reply:
I want to make every button to use the Button component so its more uniform